jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -642,6 +635,36 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Wrapper that generates a string based on various state to send to GA.
+
+      @method sendAnalytics
+      @param {arguments} arguments All arguments passed will be used.
+    */
+    sendAnalytics: function() {
+      let args = Array.apply(null, arguments);
```

Comment:
I find this hard to grok, I'd prefer using `Array.from` but this can also be done with the spread operator.
```JavaScript
args = Array.from(arguments);
  // or 
args = [...arguments];
```
Up to you